src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentForDays = 40 * days;
```

Comment:
Consider using more descriptive variable names to improve readability. For example, change `rentForDays` to `totalRent`.